velox/vector/ComplexVector.cpp

```diff
@@ -535,6 +535,22 @@ void RowVector::validate(const VectorValidateOptions& options) const {
   }
 }
 
+void RowVector::resize(vector_size_t newSize, bool setNotNull) {
+  if (newSize > length_) {
```

Comment:
shall we call BaseVector::resize(newSize, setNoNull); 
instead of lines 539-548